web-components/src/components/menu-overlay/MenuOverlay.ts

```diff
@@ -450,8 +450,8 @@ export namespace MenuOverlay {
     render() {
       return html`
         ${this.getStyles()}
-        <div aria-expanded=${this.isOpen} class="md-menu-overlay">
-          <slot name="menu-trigger"></slot>
+        <div class="md-menu-overlay">
+          <slot name="menu-trigger" aria-expanded=${this.isOpen}></slot>
           <div
```

Comment:
aria-expanded should not be put on a div with no role.
The triggering component should have the expanded property if the menu is showing